tests/api/conftest.py

```diff
@@ -9,13 +9,6 @@
 """Pytest fixtures and plugins for the API application."""
 
 import pytest
-from invenio_app.factory import create_api
-
-
-@pytest.fixture(scope="module")
-def create_app():
```

Comment:
Simply moved it up one level. Needed this fixture in order to have a more complete `app` instance with extensions (e.g. `mail` extension)